src/sentry/api/endpoints/organization_releases.py

```diff
@@ -305,6 +309,8 @@ def post(self, request, organization):
                         return Response({"projects": ["Invalid project slugs"]}, status=400)
                     projects.append(allowed_projects[slug])
 
+                new_status = result.get("status")
```

Comment:
```suggestion
                new_status = result.get("status", ReleaseStatus.OPEN)
```

Could save the `or` on line 326.

Reply:
I can't do that as we need `new_status` to be `None` for line 340, or else it will be reset to `OPEN` every time it is not specified.